.github/workflows/tests_with_database.yml

```diff
@@ -7,6 +7,67 @@ on:
   pull_request: {}
 
 jobs:
+  mysql:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - mysql:5.7
+          - mysql:8
+          - mysql:latest
+          - mariadb:10.2
+          - mariadb:10.3
+          - mariadb:10.4
+          - mariadb:10.5
+          - mariadb:10.6
+          - mariadb:10.7
+          - mariadb:latest
+
+    env:
+      NOTIFICATIONS_TESTS_DB_TYPE: mysql
+      NOTIFICATIONS_TESTS_DB: notifications
+      NOTIFICATIONS_TESTS_DB_USER: root
+      NOTIFICATIONS_TESTS_DB_PASSWORD: notifications
+      NOTIFICATIONS_TESTS_DB_HOST: 127.0.0.1
+      NOTIFICATIONS_TESTS_DB_PORT: 3306
+
+    services:
+      mysql:
+        image: ${{ matrix.image }}
+        env:
+          MYSQL_ROOT_PASSWORD: ${{ env.NOTIFICATIONS_TESTS_DB_PASSWORD }}
+          MYSQL_DATABASE: ${{ env.NOTIFICATIONS_TESTS_DB }}
+        ports:
+          - 3306:3306
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download dependencies
+        run: go get -v -t -d ./...
+
+      - name: Importing Schema
+        run: |
+          while ! mysql -h127.0.0.1 -u${{ env.NOTIFICATIONS_TESTS_DB_USER }} -p${{
+          env.NOTIFICATIONS_TESTS_DB_PASSWORD }} ${{ env.NOTIFICATIONS_TESTS_DB }} < /dev/null
+          do
+            sleep 1
+          done
+          mysql -h127.0.0.1 -u${{ env.NOTIFICATIONS_TESTS_DB_USER }} -p${{ env.NOTIFICATIONS_TESTS_DB_PASSWORD
+          }} ${{ env.NOTIFICATIONS_TESTS_DB }} < ${{ github.workspace }}/schema/mysql/schema.sql
```

Comment:
Is anything preventing the use of regular shell `\` line continuation here?

Also, these are environment variables, so shouldn't simply using `$VAR` instead of `${{ env.VAR }}` work?

And it's the same `mysql` command two times, this could be put in a variable instead, alternatively, try to use `--health-cmd` like it's already done for PostgreSQL and seems to be possible for MySQL/MariaDB as well, see https://github.com/Icinga/icinga-go-library/pull/20.